src/use_event_listener/index.jsx

```diff
@@ -1,18 +1,26 @@
-import { useEffect } from 'react';
+import { useEffect, useRef } from 'react';
 
 export const useEventListener = (
   ref,
```

Comment:
I think we should change the api of this function to take a raw element instead of a ref

Reply:
this is more flexible imo